src/public/js/contact.js

```diff
@@ -0,0 +1,108 @@
+jQuery(function ($) {
+    'use strict';
+
+    var form = $('.contact-form');
+    form.submit(function () {
+        var $this = $(this);
+        $.post("sendemail.php", $(".contact-form").serialize(), function (result) {
+            if (result.type == 'success') {
+                $this.prev().text(result.message).fadeIn().delay(3000).fadeOut();
+            }
+        });
+        return false;
+    });
+
+});
+
+// Google Map Customization
+(function () {
+
+    var map;
+
+    map = new GMaps({
+        el: '#gmap',
+        lat: 43.1580159,
+        lng: -77.6030777,
+        scrollwheel: false,
+        zoom: 14,
+        zoomControl: false,
+        panControl: false,
+        streetViewControl: false,
+        mapTypeControl: false,
+        overviewMapControl: false,
+        clickable: false
+    });
+
+    var image = 'images/map-icon.png';
+    map.addMarker({
+        lat: 43.1580159,
+        lng: -77.6030777,
+        //icon: image,
+        animation: google.maps.Animation.DROP,
+        verticalAlign: 'bottom',
+        horizontalAlign: 'center',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/Foods_Drinks_01/pullRequest?prid=7829589)
```suggestion
        horizontalAlign: "center",
```